cekit-helpers/cekit-cache-server/src/main/java/org/kie/cekit/cacher/utils/CacherUtils.java

```diff
@@ -342,4 +339,43 @@ public boolean deleteArtifact(String checksum) {
         }
     }
 
+    /**
+     * @param jarName
+     * @param zipFileName
+     * @return
+     */
+    public String detectJarVersion(String jarName, String zipFileName) {
+
+        Pattern pattern = Pattern.compile("\\d.\\d{1,2}.\\d");
+        Optional<File> zipFile = null;
```

Comment:
Why not use Optional.empty() instead of null ?

Reply:
+1